functions/helpers.bash

```diff
@@ -190,8 +190,17 @@ is_pifour_8GB() {
   totalMemory="$(awk '/MemTotal/ {print $2}' /proc/meminfo)"
   if is_pifour && [[ $totalMemory -gt 5000000 ]]; then return 0; else return 1; fi
 }
+is_cmfour() {
+  if [[ "$hw" == "cm4" ]]; then return 0; fi
+  grep -q "^Revision\\s*:\\s*[ 123][0-9a-fA-F][0-9a-fA-F][0-9a-fA-F]12[0-9a-fA-F]$" /proc/cpuinfo
+  return $?
+}
 is_pi() {
-  if is_pizero || is_pizerow || is_pione || is_cmone || is_pitwo || is_pithree || is_cmthree || is_pithreeplus || is_cmthreeplus || is_pifour; then return 0; fi
+  if is_pizero || is_pizerow || is_pione || is_cmone || is_pitwo || is_pithree || is_cmthree || is_pithreeplus || is_cmthreeplus || is_pifour || is_cmfour; then return 0; fi
+  return 1
+}
+is_cm() {}
```

Comment:
Error `is_cm() {` not `is_cm() {}`

Reply:
of course. Changed. Merge if you're ok now.